.github/workflows/repo2docker.yml

```
@@ -29,4 +29,4 @@ jobs:
 
       - name: Run repo2docker
         shell: bash -l {0}
-        run: jupyter-repo2docker --no-run --ref ${{ github.event.pull_request.head.sha }} https://github.com/${{ github.repository }}
+        run: jupyter-repo2docker --no-run --ref ${{ github.event.pull_request.head.sha }} https://github.com/${{ github.event.repository.name }}
```

Now it's trying to use https://github.com/try-gmt instead :frowning_face:

---

Try `github.event.pull_request.head.repo.full_name`? See https://github.community/t/get-the-head-repository-name-in-prs/17669

---

Ah cool, was trying to search for it. xref docs at https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request.

```suggestion
        run: jupyter-repo2docker --no-run --ref ${{ github.event.pull_request.head.sha }} https://github.com/${{ github.event.pull_request.head.repo.full_name }}
```